### PR TITLE
Feb 2020: New improvements to IPFS Bitswap

### PR DESCRIPTION
**Pull request by edgo914**

### CODE REVIEW

**Comment by jessicaschilling** [comment]
@johnnymatthews Is this one ready to go?

File: docs/concepts/bitswap.md [hunk 1]
```diff
@@ -13,23 +13,23 @@ related:
 Bitswap is a core module of IPFS for exchanging blocks of data. It directs the requesting and sending of blocks to and from other peers in the network. Bitswap is a _message-based protocol_ where all messages contain [wantlists](#wantlist) or blocks. Bitswap has a [Go implementation](https://github.com/ipfs/go-bitswap) and a [JavaScript implementation](https://github.com/ipfs/js-ipfs-bitswap).
```
Comment (johnnymatthews) [inline docs/concepts/bitswap.md]: Looks like I missed a _wantlist_ to _want-list_ change here, but can no longer edit.

File: docs/concepts/bitswap.md [hunk 2]
```diff
@@ -13,23 +13,23 @@ related:
...
-- to acquire blocks requested by the client from the network.
-- to send blocks in its possession to other peers who want them.
+- acquire blocks requested by the client from the network.
+- send blocks in its possession to other peers who want them.
 
-### Wantlist
+### Want-list
 
-A wantlist is a data structure that an IPFS node sends to the network to tell peers which blocks it wants. The wantlist contains [CIDs](https://docs-beta.ipfs.io/concepts/content-addressing/) (content-addressed identifiers that refer to particular blocks desired) along with priority and cancellation information.
+A _want-list_ is a data structure that an IPFS node sends to the network to tell peers which blocks it wants. It's a list of data that the IPFS node wants. The want-list contains [CIDs](https://docs-beta.ipfs.io/concepts/content-addressing/) (content-addressed identifiers that refer to particular blocks desired) along with priority and cancellation information.
```
Comment (johnnymatthews) [inline docs/concepts/bitswap.md]: It'd be great to see an example of a want-list here.

File: docs/concepts/bitswap.md [hunk 3]
```diff
@@ -13,23 +13,23 @@ related:
...
+-  **Data blocks.** These are blocks that have been requested (i.e., blocks in the sender's want-list, as far as the receiver is aware at the time of message receipt).
 
 ### Session
 
-IPFS content blocks are often connected to each other through a [Merkle DAG](https://docs-beta.ipfs.io/concepts/merkle-dag/). If a sender's block requests are related, a mechanism called a _Bitswap session_ in the Go implementation can optimize block requests sent to other peers. This mechanism can increase transfer speed and reduce the number of duplicate blocks on the network. Active peers are favored, as they are tracked relative to them having the requested blocks and how quickly they respond.
+IPFS content blocks are often connected through a [Merkle DAG](https://docs-beta.ipfs.io/concepts/merkle-dag/). If a sender's block requests are related, a mechanism called a _Bitswap session_ in the Go implementation can optimize block requests sent to other peers. This mechanism can increase transfer speed and reduce the number of duplicate blocks on the network. Active peers are favored, as they are tracked relative to them having the requested blocks and how quickly they respond.
```
Comment (johnnymatthews) [inline docs/concepts/bitswap.md]: Let's keep internal links relative:

```markdown
through a [Merkle DAG](/concepts/merkle-dag/).
```

**Comment by johnnymatthews** [comment]
Closing this PR. [Changes made in 112](https://github.com/ipfs/ipfs-docs-v2/pull/122).

**Pull request closed without merging**